### ISSUE TITLE
Impact Review Deck

### ISSUE THREAD

**Issue by oddball-lindsay** [department-of-veterans-affairs/va.gov-team#90345]
## Background
Need to create a deck for Impact Review using their template.  

## Tasks
- [x] Transfer content from our Oddball All Hands presentation to the Impact Review template
- [x] Update any content as needed, since the Oddball presentation was in July and Impact Review is in September
- [x] Connect with Janelle to review the deck
- [x] Inform AJ of updated slides
- [x] Add executive summary after presenting

## Acceptance Criteria
- [x] Impact Review deck is finalized and ready to present!
- [x] Impact Review has the executive summary, and has been shared with AJ

**Comment by LisaLindsey** [comment]
test

**Comment by oddball-lindsay** [comment]
@jfinnerty13 the Impact Review deck is [drafted here](https://docs.google.com/presentation/d/1Q6KEz-DLmQnRjtSmquXtHrbybE18ZGjWP4vONI0EN2w/edit#slide=id.p1).  I think it'd be good for you to see this updated format for VA, so you're not caught off guard come presentation time on Sept 5. I'm also curious about your thoughts on:
1. Slide 6: do you have any early hypotheses from last year that you'd like to include?
2. Slide 9: is it worth including July metrics for the Disability Top Navigation, or is May/June enough to demonstrate the takeaway of "we saw interest and user value in our content very quickly after launch".
3.  Slide 12: any other recommendations to the larger VA OCTO teams that you want to call out?

**Comment by oddball-lindsay** [comment]
Thank you @jfinnerty13!  

The recommendation has been updated, and I also updated the Hypothesis slide with your notes. I'm debating if the Hypotheses make sense before the Demo (as like a setup to what we developed) or after the Results (as a reflection point to what we originally hypothesized, and what we've learned since then).

The presentation is coming up quickly! Would be nice to connect on this after Weds standup if you have the time.
